server/src/operators/organization_operator.rs

```diff
@@ -629,7 +629,7 @@ pub async fn update_all_org_dataset_configs_query(
 ) -> Result<(), ServiceError> {
     let concat_configs_raw_query = sql_query(format!(
         "UPDATE datasets SET server_configuration = server_configuration || '{}' WHERE organization_id = '{}';",
-        new_config.to_string().replace('\'', "''").replace('\\', ""), org_id
+        new_config.to_string().replace('\'', "''"), org_id
```

Comment:
```suggestion
        new_config.to_string().replace('\'', "''"), org_id
```

Reply:
Shouldn't there only be one single quote in the second param spot?

Reply:
No to escape the ' in Postgres you double quote

Reply:
https://stackoverflow.com/questions/12316953/insert-text-with-single-quotes-in-postgresql

Reply:
Damn, alright